src/connect/connect.cpp

```diff
@@ -16,7 +16,16 @@
 #include <optional>
 #include <variant>
 
+using http::Connection;
+using http::ConnectionFactory;
 using http::ContentType;
+using http::Error;
+using http::HeaderOut;
+using http::HttpClient;
+using http::Method;
+using http::Request;
+using http::Response;
+using http::socket_con;
```

Comment:
In general, I'm a bit reluctant about using `using namespace`, but this might be a place where it's the lesser evil…